### DISCUSSION TITLE
Amharic Support: Thread for those interested

### DISCUSSION THREAD

**Discussion by yosiasz** [huggingface/datasets#1909] [General]
@seyyaw

Looks like you are working on some Amharic research? Anyone else you might know that could be involved?

https://discuss.huggingface.co/t/amharic-nlp-newbie-where-do-i-start/3719

Thanks

**Comment by seyyaw** [comment]
@yosiasz Yes I am working on Amharic NLP. 
In collaboration with the ICT4D center (https://bit.bdu.edu.et/ict4d/node/6), we are trying to work on multiple NLP components (models, datasets, tools) for Amharic and other Ethiopian languages.

Wiki: https://ethionlp.miraheze.org/

**Comment by ib568** [comment]
Good
